### PR TITLE
As a gitcoin user, I would like to see the responses to my post in the Notifications.

### LINKED ISSUE #5943: Add activity of Town Square to notifications
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a gitcoin user, I would like to see the responses to my post in the Notifications.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide a description of the current state, what you would like to happen, and what actually happen)
*Summary*: I would like to get the answers to my post in my notifications without having to surf all the town square or having to look at my e-mail

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
There is no notification for town square related activity

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Get notifications from someone pinging me in the town square or for responses to my post

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

Get notifications from someone pinging me in the town square or for responses to my post
- I receive a notification when someone pings me on a post
- I receive a notification when someone comments on my posts
- I receive a notification when someone pings me on some comments
##### Refers/Fixes
#5943 

##### Testing

FireShot Capture 001 - Home - Gitcoin - localhost

### CODE REVIEW

File: app/assets/v2/js/notifications.js [hunk 1]
```diff
@@ -18,9 +18,14 @@ Vue.mixin({
...
         newNotifications = newData(response.data, vm.notifications);
-        newNotifications.forEach(function(item) {
-          vm.notifications.push(item);
-        });
+
+        if (vm.notifications.length === 0) {
+          newNotifications.forEach(function(item) {
+            vm.notifications.push(item);
+          });
+        } else if (newNotifications.length > 0) {
```
Comment (octavioamu) [inline app/assets/v2/js/notifications.js]: not sure why this change, this will add the notification to the end
image

File: app/inbox/models.py [hunk 2]
```diff
@@ -55,6 +55,8 @@ class Notification(SuperModel):
         ('new_milestone', 'New Milestone'),
         ('update_milestone', 'Updated Milestone'),
         ('new_kudos', 'New Kudos'),
+        ('new_mention', 'New Mention'),
+        ('new_post_comment', 'New comment'),
```
Comment (octavioamu) [inline app/inbox/models.py]: missing new_like

**Comment by octavioamu** [comment]
Really nice and supper excited with this I have another on going PR I will integrate with this to add the notifications to the browser

File: app/inbox/models.py (hunk 2 above)
Comment (zoek1) [reply app/inbox/models.py]: Ok, i'll add it!

File: app/assets/v2/js/notifications.js (hunk 1 above)
Comment (zoek1) [reply app/assets/v2/js/notifications.js]: It's strange. I just tested the changes to avoid this behavior. 😱

Comment (octavioamu) [reply app/assets/v2/js/notifications.js]: That is because if you reload the page then the result will be on the top, but if you click the notifications to open/close, that will fetch  new notifications and then will be attached to the end.

Comment (octavioamu) [reply app/assets/v2/js/notifications.js]: just tested the new changes and now is getting on the bottom, maybe when I tested my js was cached not sure what changed because I don't remember this on the past. Maybe you are right we will need to do concat or unshift

Comment (octavioamu) [reply app/assets/v2/js/notifications.js]: ok, dont worry about this part I will do a hotfix on stable about this order but not here, in the filter part

Comment (octavioamu) [reply app/assets/v2/js/notifications.js]: fixed 68777da012cea06c64105dfa2dea77bfb01bce6e

Comment (zoek1) [reply app/assets/v2/js/notifications.js]: Thanks for the fix @octavioamu 👍
